master/usage/service-advertisement.md

```diff
@@ -12,7 +12,7 @@ to send traffic to a Kubernetes service cluster IP, and that traffic will be rou
 available endpoints for that service.
 
 You may want to enable this feature if you would like Kubernetes services to be accessible outside of the
-cluster without the need for a dedicated load balancer.
+cluster without the need for a dedicated load balancer. Please note this feature is currently limited to services having a `Type` of `ClusterIP`.
```

Comment:
I don't think this is quite right. The `Spec.Type` can be any of `ClusterIP` `NodePort` or `LoadBalancer` as per the code - https://github.com/projectcalico/confd/blob/master/pkg/backends/calico/routes.go#L224

The thing to note is that Calico currently only advertises the ClusterIP - even for LoadBalancer services.

Reply:
Thanks Casey! For me, it doesn't seem to be working for any of them so the `Spec.Type` is probably a red herring. Thanks for the review on the PR; I will abandon. If I figure this out and it can be helped with improving docs, I'll make another PR.